File: conexion_db_javascript/conexion_mongo.js
const { MongoClient } = require("mongodb");

const uri = "mongodb://localhost:27017";
const client = new MongoClient(uri);

async function conectarMongo() {
  try {
    await client.connect();
    console.log("Conexión exitosa a MongoDB Compass");

    const db = client.db("tu_base_de_datos");
    console.log("Colecciones:", await db.listCollections().toArray());
  } catch (error) {
    console.error("Error de conexión:", error);
  } finally {
    await client.close();
  }
}

conectarMongo();

File: conexion_db_java/src/conexion_mongo.java
import com.mongodb.client.MongoClients;
import com.mongodb.client.MongoClient;
import com.mongodb.client.MongoDatabase;

public class MongoDBConnection {
    public static void main(String[] args) {
        try (MongoClient client = MongoClients.create("mongodb://localhost:27017")) {
            MongoDatabase database = client.getDatabase("tu_base_de_datos");
            System.out.println("Conexión exitosa a MongoDB Compass");
            System.out.println("Colecciones: " + database.listCollectionNames());
        } catch (Exception e) {
            System.err.println("Error de conexión: " + e.getMessage());
        }
    }
}
